File: AngularKajke/src/app/shared/Models/answer-provided.ts
import { AnswerType } from './answer-type.enum';

export class AnswerProvided {
  public constructor(init?: Partial<AnswerProvided>) {
    Object.assign(this, init);
  }
  AnswerType?: AnswerType;
  QuestionNumber?: number;
  AnswerResult: string;
  AnswerNumber: number;
}

File: AngularKajke/Models/ModelClasses/AnswerProvided.cs
namespace AngularKajke.Models.ModelClasses
{
  public class AnswerProvided
  {
    private int _answerNumber;

    public AnswerType AnswerType { get; set; }
    public short QuestionNumber { get; set; }

    public int AnswerNumber
    {
      get
      {
        return _answerNumber == default(int) ? 1 : _answerNumber;
      }
      set
      {
        _answerNumber = value;
      }
    }

    public string AnswerResult { get; set; }
  }
}
